src/safeposix/syscalls/net_calls.rs

```diff
@@ -1955,6 +1993,7 @@ impl Cage {
                 continue;
```

Comment:
Based on the `if` block, shouldn't the comment for this check be changed to something like this `Ignore the fd not in readfds"?